airbyte-integrations/connectors/destination-pgvector/destination_pgvector/config.py

```diff
@@ -45,7 +45,7 @@ class PGVectorIndexingModel(BaseModel):
         examples=["AIRBYTE_DATABASE"],
     )
     default_schema: str = Field(
-        ...,
+        default="public",
```

Comment:
In Pydantic, ... is used to indicate that a field is required, while default is used to provide a default value.

These two are mutually exclusive. If a field is required (...), you cannot also provide a default value, as it creates a contradiction: the field cannot be both required and have a default.

Reply:
Just commenting if somebody is curious as I was getting `TypeError: Field() got multiple values for keyword argument 'default'`